sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/batch/BatchRequestResponseConstant.java

```diff
@@ -7,12 +7,15 @@
  * This contains all the extra constants needed for batch/bulk. This will be usefull even if Hybrid row comes in.
  * This contains all the constants we have in Backend. Any addition to backend should be added here.
  */
-public class BatchRequestResponseConstant {
+public final class BatchRequestResponseConstant {
```

Comment:
nit: should it be BatchRequestResponseConstants?

Reply:
So technically bulk is also a batching, just that it is non-txn, and internally we use batch API.

Reply:
oh, sorry, I was more around the naming part. Since this class contains multiple constants, so i think it should also reflect on the name, *Constant -> *Constants

Reply:
Oh. Apologies. Will change it. Nice catch! Thanks.

Reply:
thanks